api/filter/http/squash.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+// [#proto-status: experimental]
+
+package envoy.api.v2.filter.http;
+
+import "google/protobuf/duration.proto";
+import "validate/validate.proto";
+
+// Squash is an HTTP filter which enables Envoy to integrate with squash microservices debugger.
+// (https://github.com/solo-io/squash). When the squash filter encounters a request containing the
+// header 'x-squash-debug' it will:
+// 1. delay the incoming request.
+// 2. contact the squash server and request the creation of a debug attachment
+//    - On the squash server side, squash will attempt to attach a debugger to the services envoy
```

Comment:
Can you add this to the docs (https://1263-90675549-gh.circle-artifacts.com/0/source/generated/docs/api-v2/filter/filter.html) by updating https://github.com/envoyproxy/data-plane-api/blob/master/docs/build.sh#L28 and validate the formatting looks as expected?